content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -81,6 +81,64 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: A string representing an application environment name (for example, `prod`, `dev`) <br>
 **Description**: Adds an environment tag to all spans generated by the tracer instance.
 
+`DD_TAGS`
```

Comment:
High-level comment: All of these changes are being captured in the `Unified Service Tagging` group of configurations. Can you reorganize them to fall in the appropriate sections? E.g. `DD_DOGSTATSD_PORT` and `DD_DOGSTATSD_HOST` should go into the Agent section of configurations

Reply:
I didn't even notice the section headers 😅 Updated